### ISSUE TITLE
Be Careful or could get account suspended

### ISSUE THREAD

**Issue by PittCaleb** [dmytrostriletskyi/threads-net#27]
I wrote a script to obtain the people those you are following are following.  Goal was to help you find people to follow based off of shared interest.

After a little debugging and maybe a dozen API calls, I let the script do it's thing.  It failed after only 29 additional API calls to `threads.private_api.get_user_following`

My Instagram and Threads account are locked.  I filed for an appeal, absolutely no idea how long that takes to process.  QUite disappointed, but I suppose that's what happens when you play around with unsupported APIs :-(

**Comment by dmytrostriletskyi** [comment]
Yeah, it is true. I have mentioned this in the disclaimer — https://github.com/dmytrostriletskyi/threads-net#disclaimer

By the way, from my experience, it will take a day and you will be unsuspended.

**Comment by digfish** [comment]
Yeah. That can happen.
I have to disable the 2FA authentication in order to experiment this library. The first experiments worked well, but after some dozens of calls it started to show the same error and the last one is `'Threads' object has no attribute 'create_thread'`. I received some emails from Instagram, one for each call, for unrecognized access. I went to the Instagram settings clicked "It was me" for every unrecognized action and still the error persists. 
This a wild territory, in which you use reverse engineered stuff. You use it at your own risk.

**Comment by Abdurrafey-Siddiqui** [comment]
@dmytrostriletskyi would a solution for this be to save the login so that it doesn't make this login request every time we execute the script?
